### PR TITLE
Gh 5 policy service backed store

### PR DESCRIPTION
**Pull request by ac74475**

### CODE REVIEW

File: common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package uk.gov.gchq.palisade.policy;
```
Comment (m09526) [inline common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java]: What is the purpose of this rule which always passes everything?

Comment (m09526) [inline common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java]: Ah, I see this is used in testing code below. If this is only intended to support unit tests, would it be better off being inside the test package?

Comment (m09526) [inline common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java]: ---

Comment (ac74475) [reply common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java]: So it is in the common test jar due to it needing to be distributed with all the data services for the docker/multi JVM examples

**Pull request merged**